verification/curator-service/api/src/clients/email-client.ts

```diff
@@ -67,23 +58,26 @@ export default class EmailClient {
      * @param subject - Subject of the email.
      * @param text - Body/content of the email.
      */
-    send = (
+    send(
         addressees: string[],
         subject: string,
         text: string,
-    ): Promise<SentMessageInfo> => {
+    ): Promise<SentMessageInfo> {
+        if (this.transport === undefined) {
+            throw new Error('Client must be initialized prior to use.');
```

Comment:
nit but best practice to explicitely say what's needed (did you forget to call initialize()?)

Reply:
SG, done.